src/main/java/net/sf/jabref/logic/util/io/SimpleFileListEntry.java

```diff
@@ -0,0 +1,94 @@
+/*  Copyright (C) 2015 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.logic.util.io;
+
+import java.util.List;
+
+/**
+ * This class represents a simplified version of the file link for a Bibtex entry.
+ */
+public class SimpleFileListEntry {
```

Comment:
Except for `getStringRepresentation` (and the fact that it relies on static methods rather than keeping things as a class). I'll wait and see what the other guys have to say about it. I could in fact think that it may make sense to provide the file field directly to the constructor. May not change the fundamental discussion, but from other perspectives one probably would like that anyway.